.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,26 @@
+---
+name: User Story
+about: Create an issue in the team's standard format
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+#### User Story
+As a issue creator
+
+I want to be able to create issues
+
+So that I can see my issue resolved
+
+#### Acceptance Criteria: 
+- [ ] The issue exists in the list of issues
```

Comment:
I think the "meta" examples here and on the User story are pretty confusing

Reply:
These are just here to give us a formatting jump start and have been approved by Carrie. What would you propose as an alternative?

Reply:
- [ ] This condition that must be met for the card to be complete
- [ ] E.g. Reaper can do a thing

Exact wording can be whatever but I think having explanations or examples makes it clearer what part of the template is part of the format and what part should be replaced with details specific to the card.